React/package.json

```
@@ -0,0 +1,41 @@
+{
+  "name": "test-app",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.11.4",
+    "@testing-library/react": "^11.1.0",
+    "@testing-library/user-event": "^12.1.10",
+    "devextreme": "^22.1.5",
+    "devextreme-react": "^22.1.5",
+    "react": "^16.14.0",
+    "react-dom": "^16.14.0",
+    "react-scripts": "4.0.2",
+    "rrule": "^2.7.1",
+    "web-vitals": "^1.0.1"
+  },
```

Let's update dependencies to more actual:

```suggestion
"dependencies": {
    "@testing-library/jest-dom": "^5.16.5",
    "@testing-library/react": "^13.4.0",
    "@testing-library/user-event": "^14.4.3",
    "devextreme": "^22.2.4",
    "devextreme-react": "^22.2.4",
    "react": "^18.2.0",
    "react-dom": "^18.2.0",
    "react-scripts": "5.0.1",
    "rrule": "^2.7.1",
    "web-vitals": "^3.1.1"
  },
```

And regenerate package-lock.json